docs/Session Notes/Session 05.md

```diff
@@ -0,0 +1,19 @@
+# Session 05
+
+Arriving in [[Camp Righteous]], the party encountered new allies. Stranded by [[Aremag]] and taking canoes down river, they were seperated from [[Artus Cimber]] and only arrived right before [[Locke Handcock]], [[Brysh]], [[Xandala Cimber]] and folks arrived.
+
+[[Xandala Cimber]] had a bit of a heart to heart with the friends of her father in this new party.
+
+Cousins [[Yosei]] and [[Sylvos]] were taught some "archeologist" adventurer techniques by throwing ball bearings at an out house. [[Yosei]] lost one in the grass, but he also found a dead body in the outhoust with a holy symbol of Mystra. It's about 5 days old.
+
+At the center of the camp was a large status of [[Man and Crocodile]]. Inside were several traps:
+- A 20ft pit trap activated by a creature standing on it.
+- Magic blades which appear when a creature is not carrying another creature.
+- Up a 7ft wall there's a 4x4 grid of squares. Stepping on the wrong squares causes fire to spew out.
+- Up another 7ft wall was a door with a glyph of warding. Two creatures must push on it to open it.
```

Comment:
It was not that two of us were pushing on it, it was that there were 4 lights and we had to push them at the same time wasnt it?